doc/changes.md

```diff
@@ -1,5 +1,96 @@
 # Changelog
 
+## Release v0.6.0 (2022/03/14)
+
+[![Digital Object Identifier for PyGMT v0.6.0](https://zenodo.org/badge/DOI/10.5281/zenodo.6349217.svg)](https://doi.org/10.5281/zenodo.6349217)
+
+### Highlights
+
+* 🎉 **Sixth minor release of PyGMT** 🎉
+* New inline examples for 13 functions!
+* Minimum required GMT version is now 6.3.0 ([#1649](https://github.com/GenericMappingTools/pygmt/pull/1649))
```

Comment:
Per @willschlitzer's suggestion at https://github.com/GenericMappingTools/pygmt/pull/1808#issuecomment-1066631155. Feel free to edit to be more shorter.

```suggestion
* Single `pygmt.datasets.load_sample_data` function for loading any sample dataset ([#1685](https://github.com/GenericMappingTools/pygmt/pull/1685))
* Minimum required GMT version is now 6.3.0 ([#1649](https://github.com/GenericMappingTools/pygmt/pull/1649))
```